src/common/classes/ErrorAdapter.test.ts

```diff
@@ -0,0 +1,22 @@
+import ErrorAdapter from './ErrorAdapter';
+
+import { ApiErrors, ErrorTest } from '~common/enums/Error';
+
+describe('ErrorAdapter', () => {
+  it('should return the message based on the information sent', () => {
+    const type = 'API_ERROR';
+    expect(new ErrorAdapter({ type }).getByType()).toBe(ApiErrors[type]);
```

Comment:
Isso é algum mock para testes com a API? Eu não entendi bem, por isso não sei se é caso de seguir o padrão abaixo:
```suggestion
    const type = 'API_ERROR';
    
    expect(new ErrorAdapter({ type }).getByType()).toBe(ApiErrors[type]);
```

Reply:
É somente pra ele pegar o valor de forma mais fácil e não ficar tudo na mesma linha, prefiro separar os valores usados no teste.